docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md

```diff
@@ -0,0 +1,81 @@
+# Dasharo compatibility: Sign of life
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+## SOL001.001 Detection of Dasharo firmware and Dasharo EC firmware
```

Comment:
@Pokisiekk I think we should select appropriate names for these two test cases. 

At this point, as if I had glanced at the name alone, I would have suspected that the test requires the execution of some commands that will return output containing information about the firmware and EC versions.

So I would suggest:
1. SOL001.001 SOL string shows Dasharo firmware and EC version
2. SOL002.001 SOL string shows information about proprietary EC

Reply:
Updated